quipucords/quipucords/settings.py

```diff
@@ -162,13 +162,26 @@ def is_int(value):
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
 # Database Management System could be 'sqlite' or 'postgresql'
-QPC_DBMS = os.getenv('QPC_DBMS', 'sqlite').lower()
+QPC_DBMS = os.getenv('QPC_DBMS', 'postgres').lower()
 
-if QPC_DBMS == 'postgres':
+if QPC_DBMS == 'sqlite':
+    # If user enters an invalid QPC_DBMS, use default sqlite
```

Comment:
we should remove or move and edit this comment because the default is no longer sqlite